packages/backend/src/plugins/techInsights.ts

```diff
@@ -38,41 +38,10 @@ export default async function createPlugin({
     database,
     discovery,
     factRetrievers: [
-      createFactRetrieverRegistration('5 4 * * 6', {
-        // Example cron, At 04:05 on Saturday.
-        id: 'testRetriever',
-        version: '1.1.2',
-        entityFilter: [{ kind: 'component' }], // EntityFilter to be used in the future (creating checks, graphs etc.) to figure out which entities this fact retrieves data for.
-        schema: {
-          examplenumberfact: {
-            type: 'integer',
-            description: 'Example fact returning a number',
-          },
-        },
-        handler: async _ctx => {
-          const catalogClient = new CatalogClient({
-            discoveryApi: discovery,
-          });
-          const entities = await catalogClient.getEntities({
-            filter: [{ kind: 'component' }],
-          });
-
-          return Promise.resolve(
-            entities.items.map(it => {
-              return {
-                entity: {
-                  namespace: it.metadata.namespace!!,
-                  kind: it.kind,
-                  name: it.metadata.name,
-                },
-                facts: {
-                  examplenumberfact: 2,
-                },
-              };
-            }),
-          );
-        },
-      }),
+      createFactRetrieverRegistration(
+        '* * * * *', // Example cron, every minute
+        createCatalogFactRetriever(),
+      ),
     ],
     factCheckerFactory: new JsonRulesEngineFactCheckerFactory({
       checks: [
```

Comment:
I also plan to update the default check to leverage the new facts from the above retriever.